File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/HeartOfAzeroth.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class HeartOfAzeroth {

    @SerializedName("essences")
    @Expose
    public List<Essence> essences = null;
    @SerializedName("level")
    @Expose
    public Integer level;
    @SerializedName("progress")
    @Expose
    public Float progress;

}

File: app/src/main/java/fr/jbme/raiderioapp/model/RIOCharacter/HeartOfAzeroth.kt
package fr.jbme.raiderioapp.model.RIOCharacter

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class HeartOfAzeroth(
    @SerializedName("essences")
    @Expose
    var essences: List<Essence>? = null,

    @SerializedName("level")
    @Expose
    var level: Int? = null,

    @SerializedName("progress")
    @Expose
    var progress: Float? = null
)